<launch>
	<node name = "slam_node" pkg = "ohm_tsd_slam" type = "slam_node" output = "screen">
		<param name = "laser_topic" type = "string" value = "simon/scan"/>
		<param name = "object_inflation_factor" type = "int" value = "2"/>
		<param name = "x_off_factor" type = "double" value = "0.2"/>
		<param name = "y_off_factor" type = "double" value = "0.5"/>
    <param name = "cell_octave_factor" type = "int" value ="10" />
    <param name = "cellsize" type = "double" value = "0.025" />
    <param name = "range_filter" type = "bool" value = "true" />
    <param name = "min_range" type = "double" value = "0.01" />
    <param name = "max_range" type = "double" value = "10.0" />
    <param name = "occ_grid_time_interval" type = "double" value = "2.0" />
    <param name = "loop_rate" type = "double" value = "40.0" />
    <param name = "sensor_static_offset_x" type = "double" value = "-0.19" />
    <param name = "pose_topic" type = "string" value = "pose" />
    <param name = "tf_base_frame" type = "string" value = "/map" />
    <param name = "tf_child_frame" type = "string" value = "base_footprint" />
	</node>
</launch>
